test/libsolidity/SemanticTest.cpp

```diff
@@ -241,7 +241,7 @@ TestCase::TestResult SemanticTest::run(ostream& _stream, string const& _linePref
 
 void SemanticTest::printSource(ostream& _stream, string const& _linePrefix, bool) const
 {
-	stringstream stream(m_source);
+	stringstream stream(m_source.sourceMap.at(m_source.mainSourceFile));
```

Comment:
Shouldn't this print all the source files? Currently it prints only the main one.